main.py

```diff
@@ -176,7 +175,8 @@ def write_metric_calculations_to_file(data_filepath, metric_calculations, should
   return False
 
 
-def build_filename(resource_type, outpath, date, duration, site, client_provider, metric):
+def build_filename(resource_type, outpath, date, duration, site,
+                    client_provider, client_country, metric):
```

Comment:
Can we move this to utils.py and add tests? I'd like to trim down main.py as much as possible and reduce the number of functions that are missing unit test coverage.